srcpkgs/buku/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'buku'
 pkgname=buku
-version=3.8
+version=4.0
 revision=2
 noarch=yes
 wrksrc="Buku-${version}"
+depends="python3-urllib3 python3-BeautifulSoup4"
 checkdepends="python3-BeautifulSoup4 python3-cryptography python3-hypothesis
```

Comment:
`vcrpy` is missing to `checkdepends`.
Keep tests functional or stop adding `do_check()` to Python modules.
This can be done in a separate PR.

Reply:
I prepared a another PR with the checkroutine deleted.